beta/nncf/tensorflow/graph/converter.py

```diff
@@ -93,19 +96,25 @@ def get_graph_to_layer_var_names_map(concrete_fun):
     graph_to_layer_names_map = {} if use_graph_var_names else get_graph_to_layer_var_names_map(concr_fn)
 
     nncf_graph = NNCFGraph()
+
+    graphdef_node_name_to_id = {}  # type: Dict[str, int]
+
     for node in nodes:
-        nncf_graph.add_node(
-            graph_to_layer_names_map.get(node.name, node.name),
-            type=node.op,
-            dtype=node.attr['dtype'],
-            metatype=get_op_metatype(node.op))
+        nncf_node = nncf_graph.add_nncf_node(node_name=graph_to_layer_names_map.get(node.name, node.name),
+                                             node_type=node.op,
+                                             node_metatype=get_op_metatype(node.op))
+        graphdef_node_name_to_id[node.name] = nncf_node.node_id
 
     for node in nodes:
-        for input_node in node.input:
-            node_name = graph_to_layer_names_map.get(node.name, node.name)
-            input_node_name = graph_to_layer_names_map.get(input_node, input_node)
-            if input_node_name in nncf_graph.get_all_node_keys():
-                nncf_graph.add_edge(input_node_name, node_name)
+        dtype = Dtype.FLOAT if node.attr['dtype'].type == 1 else Dtype.INTEGER
+        for idx, input_node in enumerate(node.input):
+            nncf_node_id = graphdef_node_name_to_id[node.name]
+            if input_node in graphdef_node_name_to_id:
+                input_nncf_node_id = graphdef_node_name_to_id[input_node]
+                nncf_graph.add_edge_between_nncf_nodes(input_nncf_node_id,
+                                                       nncf_node_id,
+                                                       tensor_shape=None,  # TODO: fix
```

Comment:
Please, add TODO according to the Style Guide.

Reply:
Done